syncthing-quick-status.sh

```diff
@@ -15,8 +15,10 @@ done
 
 if [[ -z $SYNCTHING_API_KEY ]]; then
 	: "${SYNCTHING_CONFIG_FILE:="$HOME/.config/syncthing/config.xml"}"
-	apikey_regex='^\s+<apikey>([^<]+)</apikey>$'
-	apikey_line="$(grep -E "$apikey_regex" "$SYNCTHING_CONFIG_FILE")"
+       # MacOS stores config.xml in a different path:
+       uname -o | grep -q Darwin && SYNCTHING_CONFIG_FILE="$HOME/Library/Application Support/Syncthing/config.xml"
```

Comment:
There's a little too much indentation here!
Also, if I'm on a Mac, with your code I cannot override SYNCTHING_CONFIG_FILE.

A better way to do this, while conserving the "overridibility", would be to set a DEFAULT_SYNCTHING_CONFIG_FILE, depending on the os, then use it instead of the Linux default above.